package-lock.json

```diff
@@ -93,7 +93,7 @@
     "clients/web": {
```

Comment:
Just wanna confirm that all these changes to `package-lock.json` are intentional. This PR doesn't contain any newly installed packages.

Reply:
No changes to package.json, so this should be fine